src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -818,6 +836,40 @@ uct_ib_mlx5_devx_open_device(struct ibv_device *ibv_device)
 
 static uct_ib_md_ops_t uct_ib_mlx5_devx_md_ops;
 
+static ucs_status_t uct_ib_mlx5_devx_init_flush_remote(uct_ib_mlx5_md_t *md)
+{
+    intptr_t addr = (intptr_t)md->zero_buf & ~(UCT_IB_MD_MAX_MR_SIZE - 1);
+    /* FW requires indirect atomic MR addr and length to be aligned
+     * to max supported atomic argument size */
+    size_t length = ucs_align_up(ucs_get_page_size() +
+                                 (intptr_t)md->zero_buf - addr,
+                                 ucs_get_page_size());
+    int list_size = ucs_div_round_up(length, UCT_IB_MD_MAX_MR_SIZE);
```

Comment:
I would expect it will always be 1 element since flush memory region should be 8 bytes (or maybe even 1 byte)